frontend/next.config.js

```diff
@@ -1,32 +1,15 @@
-const dotenv = require('dotenv');
-
-const env = dotenv.config({ path: '../.env' });
-if (env.error) {
-  throw env.error;
-}
-
-// Add 'HOSTNAME' to environment
-if (!('HOSTNAME' in env.parsed)) {
-  env.parsed['HOSTNAME'] = (
-    env.parsed['ENVIRONMENT'] === "dev"
-      ? "http://localhost:8000"
-      : "https://modularhistory"
-  );
-} else {
-  throw new Error(".env var 'HOSTNAME' is already defined");
-}
+// https://nextjs.org/docs/api-reference/next.config.js/introduction
 
 module.exports = {
   // Delegate static file compression to Nginx in production
-  compress: env.parsed['ENVIRONMENT'] === "prod" ? false : true,
-  env: env.parsed,
+  compress: process.env.ENVIRONMENT === "prod" ? false : true,
```

Comment:
Do we want compression enabled in development? It seems like it would only be beneficial for network speeds, which are irrelevant for local dev containers.

Reply:
Hmm, I think I'd like for compression to be enabled one way or another in dev, if it's enabled one way or another in production. In production, it's handled by Nginx. In dev, I guess I'm fine with having it done by Next.js. (It's enabled by default, so the point here is just to disable it in prod where it's handled by Nginx.)